docs/UserGuide.md

```diff
@@ -112,13 +112,15 @@ Examples:
 
 ### Locating tasks by name: `find`
 
-Finds tasks whose titles best match the user input string.
+Finds tasks whose attribute best match the user input string.
 
-Format: `find SUBSTRING`
+Format: `find n/{NAME}` OR `find d/{DESCRIPTION}` OR `find t/{TAG}...`
 
 * The search is case-insensitive. e.g `book` will match `Book`
-* Only the title is searched.
+* Use only 1 attribute at a time.
 * Substrings will be matched e.g. `book` will match `Books`
+* For tags, as long as one tag matches with one of the tags you are searching for, it will be considered matched.
+  * e.g. `find t/urgent t/important` will match with tags `t/urgent t/math t/hard` since it has `urgent`.
 
 Examples:
 * `find book` returns `read book` and `return books`
```

Comment:
Would it be clearer to also include some examples where we pass in more than 1 word as keyword?

Reply:
I'm not sure what you mean, do you mean something like `find t/very urgent`?

Reply:
Yep! It might be useful for users to know that they can do more with our program :)

Reply:
Ok! Thanks for the suggestion, will update as suggested.